docs/cn/404.html

```diff
@@ -1,23 +1,20 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
+<!DOCTYPE html><html lang="en"><head>
```

Comment:
en or cn?

Reply:
I guess it should be `cn` but I don't know how to change that.